compiler/cpp/src/generate/t_java_generator.cc

```diff
@@ -5139,8 +5215,11 @@ THRIFT_REGISTER_GENERATOR(
     "    reuse-objects:   Data objects will not be allocated, but existing instances will be used "
     "(read and write).\n"
     "    sorted_containers:\n"
-    "                     Use TreeSet/TreeMap instead of HashSet/HashMap as a implementation of "
+    "                     Use TreeSet/TreeMap instead of HashSet/HashMap as an implementation of "
     "set/map.\n"
+    "    enum_containers:\n"
```

Comment:
Not so sure if we need a new option at all. The default assumptions for Thrift containers are described at https://thrift.apache.org/docs/types . If the patch does not break compatibility and on top is more efficient, why not make it the default? $0,02